tests/panels/charts/xy/test_xy.py

```
@@ -333,3 +341,219 @@ async def test_area_percentage_chart() -> None:
+    )
+
+
+async def test_reference_line_with_value_object() -> None:
+    """Test reference line with XYReferenceLineValue instead of float."""
+    ref_line = XYReferenceLine(
+        label='Baseline',
+        value=XYReferenceLineValue(value=100.0),
+    )
+
+    layer_id = 'test-layer-id-2'
+    ref_layer, ref_column = compile_reference_line_layer(ref_line, layer_id)
```

Ruff (0.14.10)

Unpacked variable `ref_layer` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)